lizmap/modules/lizmap/controllers/edition.classic.php

```diff
@@ -541,14 +541,19 @@ public function editFeature()
             $title = 'No title';
         }
 
+        // Prepare template
+        $attributeEditorForm = $qgisForm->getAttributesEditorForm();
+        $form = $qgisForm->getForm();
+
         // Use template to create html form content
         $tpl = new jTpl();
-        $tpl->assign('attributeEditorForm', $qgisForm->getAttributesEditorForm());
+        $tpl->assign('attributeEditorForm', $attributeEditorForm);
         $tpl->assign('fieldNames', $qgisForm->getFieldNames());
         $tpl->assign('title', $title);
-        $tpl->assign('form', $qgisForm->getForm());
-        $tpl->assign('formPlugins', $qgisForm->getFormPlugins());
+        $tpl->assign('form', $form);
+        $tpl->assign('formPlugins', $qgisForm->getFormPlugins());HEAD
```

Comment:
```suggestion
        $tpl->assign('formPlugins', $qgisForm->getFormPlugins());
```
You have lost your HEAD :-)

Reply:
@rldhont HEAD still lost :)

Reply:
Fixed thanks